fmt/README.md

```diff
@@ -215,3 +97,73 @@ contract HelloWorld {
 
 event Greet(string indexed name);
 ```
+
+### Configuration
+
+Formatter supports multiple configuration options defined in `FormatterConfig`.
+
+| Option                            | Default  | Description          
+| --------------------------------- | -------  | ---------------------
+| line_length                       | 120      | Maximum line length where formatter will try to wrap the line
+| tab_width                         | 4        | Number of spaces per indentation level
+| bracket_spacing                   | false    | Print spaces between brackets
+| int_types                         | long     | Style of uint/int256 types. Available options: `long`, `short`, `preserve`
+| func_attrs_with_params_multiline  | true     | If function parameters are multiline then always put the function attributes on separate lines
+| quote_style                       | double   | Style of quotation marks. Available options: `double`, `single`, `preserve`
+| number_underscore                 | preserve | Style of underscores in number literals. Available options: `remove`, `thousands`, `preserve`
```

Comment:
I think these might not all exist in the docs? https://book.getfoundry.sh/reference/config?highlight=fmt#formatter

Reply:
yup, `int_types`, `number_underscore` & `func_attrs_with_params_multiline` are missing

Reply:
@gakonst fixed https://github.com/foundry-rs/book/pull/534/files